src/ledger/pathfind.ts

```diff
@@ -23,7 +23,7 @@ function addParams(request: PathFindRequest, result: RippledPathsResponse
 function requestPathFind(connection: Connection, pathfind: PathFind
 ): Promise<RippledPathsResponse> {
   const destinationAmount: Amount = _.assign(
-    {value: '-1'},
+    {value: dropsToXrp('-1')},
```

Comment:
What do you think about moving this change to https://github.com/ripple/ripple-lib/pull/930?

Reply:
Good call